qa/suites/rados/singleton-bluestore/all/cephtool.yaml

```diff
@@ -22,10 +22,10 @@ tasks:
     - failsafe engaged, dropping updates
     - failsafe disengaged, no longer dropping updates
     - overall HEALTH_
-    - (OSDMAP_FLAGS)
-    - (OSD_
-    - (PG_
-    - (SMALLER_PG_NUM)
+    - \(OSDMAP_FLAGS\)
```

Comment:
it's already fixed in 918921ab2f939c8b52c43a10bf0266347a2bdcf3 . and more of them were added in ee4acb6e1ff7458ceaefdb288cbcb158c6a3bed3.